ci/jenkins/release

```diff
@@ -0,0 +1,160 @@
+                        sh "cd plugins && curl -k sftp://rtpgsa.ibm.com/projects/c/certsign/liberty_tools/signed/${LAST_COMMIT}/${LS_JAR} -o ${LS_JAR} --user '$user:$password'"
+                        sh "cd plugins && curl -k sftp://rtpgsa.ibm.com/projects/c/certsign/liberty_tools/signed/${LAST_COMMIT}/${UI_JAR} -o ${UI_JAR} --user '$user:$password'"
+                        sh "cd plugins && curl -k sftp://rtpgsa.ibm.com/projects/c/certsign/liberty_tools/signed/${LAST_COMMIT}/${PRODUCT_JAR} -o ${PRODUCT_JAR} --user '$user:$password'"
+                    }  
+                }
+                
+                dir('liberty-tools-eclipse') {
+                    // Run Tycho plugin to recreate artifacts conten
+                    sh "mvn -pl releng tycho-p2-repository:fix-artifacts-metadata -Drepository.home=io.openliberty.tools.update"
+                }
+            }
+        }
+        
+        // The Install stage was added to verify the user will not be prompted with "Accepting unsigned content"; however, this stage
+        // passes whether or not the plugin jars are signed. Commenting this out until we have a better way to verify.
+        
+        // stage('Install') {
+        //     steps {
+        //         dir('install-test') {
+        //             sh "curl https://download.eclipse.org/technology/epp/downloads/release/2022-09/R/eclipse-jee-2022-09-R-linux-gtk-x86_64.tar.gz --output eclipse.tar.gz"
+        //             sh "tar -xf eclipse.tar.gz"
+        //         }
+        //         dir('install-test/eclipse') {
+        //             script {
+        //                 WORKING_DIR = sh (
+        //                                 script: "pwd",
+        //                                 returnStdout: true
+        //                             ).trim()
+        //             }
+        //             sh "./eclipse -nosplash -application org.eclipse.equinox.p2.director -repository file:///${WORKING_DIR}/../../liberty-tools-eclipse/releng/io.openliberty.tools.update/target/repository,https://raw.githubusercontent.com/eclipse/lsp4jakarta/build-p2-update-site-0.0.1,http://download.eclipse.org/lsp4mp/releases/0.5.0/repository/,https://download.eclipse.org/releases/2022-09/,https://download.eclipse.org/jdtls/snapshots/repository/latest/ -installIU io.openliberty.tools.eclipse.feature.group > install-plugin-console-output.txt"
+        //             archiveArtifacts artifacts: 'install-plugin-console-output.txt', fingerprint: true
+        //         }
+        //     }
+        // }
+                
+        stage('Export') {
+            steps {
+                dir('liberty-tools-eclipse/releng/io.openliberty.tools.update/target') {
+                    
+                    // Zip repository directory
+                    sh "cd repository && zip -r repository.zip * && mv repository.zip .."
+                    
+                    archiveArtifacts artifacts: 'repository/**/*.*', fingerprint: true
```

Comment:
Maybe a dumb question but what's 'archiveArtifacts' ?  Is that a Jenkins command?

Reply:
Yup - that "saves" the specified files for that build